src/i18n/locales/en.json

```diff
@@ -3,6 +3,8 @@
 		"home": "{FilesName} • Home",
 		"docs_home": "{FilesName} • Docs",
 		"docs_page": "{FilesName} • Docs - {title}",
+		"blog_home": "{FilesName} • Blog",
+		"blog_page": "{FilesName} • {title}",
```

Comment:
@TheOnlyTails would you prefer it if this string were more consistent with the docs string?
e.g.:
```json
		"blog_page": "{FilesName} • Blog - {title}",
```